tests/e2e/README.md

```diff
@@ -64,6 +64,10 @@ Start tests based on the database container
 TEST_MSSQL_HOST=localhost:1433 TEST_MSSQL_DBNAME=gitea_test TEST_MSSQL_USERNAME=sa TEST_MSSQL_PASSWORD=MwantsaSecurePassword1 make test-e2e-mssql
 ```
 
+## Debug
+
+Set `PLAYWRIGHT_DEBUG=1` to enable [visual debugging](https://playwright.dev/docs/debug#run-in-debug-mode-1).
```

Comment:
```suggestion
Set the environment variable `PLAYWRIGHT_DEBUG` to enable [visual debugging](https://playwright.dev/docs/debug#run-in-debug-mode-1).
```
Alternatively, we can even require the variable to be truthy (→ `strconv.ParseBool`).